src/interfaces/HposInterface.ts

```diff
@@ -23,9 +23,36 @@ interface HposInterface {
   getUnpaidInvoices: () => Promise<HposHolochainCallResponse>
   getRedemptionHistory: () => Promise<HposHolochainCallResponse>
   getCoreAppVersion: () => Promise<CoreAppVersion>
+  redeemHoloFuel: (payload: RedeemHoloFuelPayload) => Promise<HposHolochainCallResponse>
   HPOS_API_URL: string
 }
 
+interface RedeemHoloFuelPayload {
+  amount: string
+  note: string
+}
+
+export interface RedemptionTransaction {
+  id: string
+  amount: number
+  fee: string
+  completed_date: number
+  created_date: number
+  transaction_type: string
+  counterparty: string
+  direction: string
+  status: string
+  note?: string // The `eth_hash` will be stored here in Redemption Transactions
+  url?: string
+  expiration_date?: number
+}
+
+interface ReserveSettingsResponse {
+  hash: string
```

Comment:
```suggestion
  reserve_id: string
```

Updating to match implementation changes.